DEPLOYING.md

```diff
@@ -0,0 +1,126 @@
+# Deploying Philanthropy Data Commons software
+
+This document guides one through deployment to production of the back-end
+Philanthropy Data Commons software to run at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has [its own separate
+process][1].
+
+What gets deployed? Everything specified in a [docker compose yaml file][2].
+This includes the PDC service ("web") and other services upon which it depends.
+
+How do they get deployed? A [bash script running under cron][3] gets the new
+compose file, pulls new components (images) using the new compose file, stops
+the old services (containers) using the old compose file, and starts the new
+services (containers) using the new compose file.
+
+How does the deploy script know how what to deploy? It [looks inside a file][4].
```

Comment:
I took the "what file" comment as a suggestion added the value of `TAG_FILE`.

As for Q&A I am still in favor of the Q&A but obviously this document is not for me. It's not necessarily for you either, so I am curious what other folks think. As for why I care about the Q&A, I think the questions are relevant to all deployment pipelines, and the answers here are specific to this deployment pipeline, and the answers here may even change for this pipeline. But the format wouldn't change just because the implementation or answers changed.

@smpsnr or @kfogel? Q&A or simple declarations or something else?

Reply:
I removed the questions.